doc/sphinx/source/releasenotes.rst

```diff
@@ -4,6 +4,22 @@ Changes/Release Notes
 On this page we provide a summary of the main API changes, new features and examples
 for each release of libCEED.
 
+.. _master:
```

Comment:
@jedbrown, @valeriabarra, what about maintaining a running log of big updates to master since the last release? It would make future release notes easier to generate and highlight API changes in master.

Reply:
Yes, I meant to ask for us to start this practice. I would try to organize similarly to the v0.6 release notes section. Whatever you put there will show up on the website
https://libceed.readthedocs.io/en/latest/releasenotes/